Source/SIMPLib/CoreFilters/RenameAttributeArray.cpp

```diff
@@ -118,44 +117,41 @@ void RenameAttributeArray::dataCheck()
   }
 
   AttributeMatrix::Pointer attrMat = getDataContainerArray()->getPrereqAttributeMatrixFromPath<AbstractFilter>(this, getSelectedArrayPath(), -301);
-  if(getErrorCondition() < 0)
+  if(getErrorCode() < 0)
   {
     return;
   }
 
   RenameErrorCodes code = attrMat->renameAttributeArray(daName, m_NewArrayName, false);
   switch(code)
   {
-  case OLD_DOES_NOT_EXIST:
-  {
-    setErrorCondition(-11016);
-    QString ss = QObject::tr("A DataArray with the name '%1' was not found in the AttributeMatrix").arg(daName);
-    notifyErrorMessage(getHumanLabel(), ss, getErrorCondition());
-    return;
-  }
-  case NEW_EXISTS:
-  {
-    setErrorCondition(-11017);
-    QString ss = QObject::tr("A DataArray with the name '%1' already exists in the AttributeMatrix").arg(m_NewArrayName);
-    notifyErrorMessage(getHumanLabel(), ss, getErrorCondition());
-    return;
-  }
-  case RenameErrorCodes::SAME_PATH:
-  {
-    setErrorCondition(-11018);
-    QString ss = QObject::tr("The new and original DataArray names cannot be identical");
-    notifyErrorMessage(getHumanLabel(), ss, getErrorCondition());
-    return;
-  }
-  case SUCCESS:
-  {
-    setErrorCondition(0);
-    setWarningCondition(0);
-    DataArrayPath newPath = getSelectedArrayPath();
-    newPath.setDataArrayName(m_NewArrayName);
-    addPathRename(getSelectedArrayPath(), newPath);
-    return;
-  }
+    case OLD_DOES_NOT_EXIST:
+    {
+      QString ss = QObject::tr("A DataArray with the name '%1' was not found in the AttributeMatrix").arg(daName);
+      setErrorCondition(-11016, ss);
+      return;
+    }
+    case NEW_EXISTS:
+    {
+      QString ss = QObject::tr("A DataArray with the name '%1' already exists in the AttributeMatrix").arg(m_NewArrayName);
+      setErrorCondition(-11017, ss);
+      return;
+    }
+    case RenameErrorCodes::SAME_PATH:
+    {
+      QString ss = QObject::tr("The new and original DataArray names cannot be identical");
+      setErrorCondition(-11018, ss);
+      return;
+    }
+    case SUCCESS:
+    {
+      clearErrorCondition();
+      clearWarningCondition();
+      DataArrayPath newPath = getSelectedArrayPath();
+      newPath.setDataArrayName(m_NewArrayName);
+      addPathRename(getSelectedArrayPath(), newPath);
+      return;
+    }
```

Comment:
Something happened to your formatting here.  None of these cases should be indented, at least not compared to the preceding switch statement.